src/utils.py

```diff
@@ -99,3 +110,37 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def get_semver_command(body):
+    match = re.match(
+        rf".*\B/(?P<{COMMAND}>{COMMAND_MAJOR}|{COMMAND_MINOR}|{COMMAND_PATCH})\b",
+        body,
+        re.IGNORECASE,
+    )
+    if match:
+        return match.group(COMMAND).lower()
+
+
+def semver_label_in_pull(pull):
+    return any(
+        label in [LABEL_SEMVER_MAJOR, LABEL_SEMVER_MINOR, LABEL_SEMVER_PATCH]
+        for label in get_labels(pull=pull)
+    )
+
+
+def add_remove_labels(pull, add_labels, remove_labels):
```

Comment:
The naming is a bit confusing
I think `labels_to_add` and `labels_to_remove` describe it better.